Task_DEV_4/Task_DEV_4/Triangle.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+
+namespace Task_DEV_4
+{
+    class Triangle : Figure
+    {
+        private const string ERROR_MESSAGE = "Side cannot be less equals than 0.";
+        private const string ERROR_MESSAGE2 = "The sum of the two sides cannot be less than the third side.";
+
+        public double BaseSide { get; set; }
+        public double FirstSide { get; set; }
+        public double SecondSide { get; set; }
+
+        public Triangle(string color, string name, double baseSide, double firstSide, double secondSide) : base(color, name)
+        {
+            BaseSide = baseSide;
+            FirstSide = firstSide;
+            SecondSide = secondSide;
+            CheckForExistance();
+        }
+
+        public override double CalculateArea()
+        {
+            double perimeter = BaseSide + FirstSide + SecondSide;
```

Comment:
you can reuse CalculatePerimeter() method for perimeter variable.

Reply:
Done.